python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -55,10 +55,9 @@ class MultiPostProcessor(IPostProcessor):
         'process_objects': ComponentItem('postprocessor', defvalue=[], hidden=1, doc='A list of Processors to run', sequence=1)
     })
 
-    def __init__(self):
+    def __init__(self, *args):
         super(MultiPostProcessor, self).__init__()
 
-    def __construct__(self, args):
         if len(args) == 1 or len(args) > 1 and isType(args, (GangaList, list)):
```

Comment:
This didn't expose any problems and frankly I was just looking to remove the `__construct__` method which shouldn't be called. I can fixup this code but I'm now worried why nothing has broken here in testing?